Jenkinsfile

```
@@ -158,7 +158,14 @@ pipeline {
             if (params.WORKLOAD == "node-density" || params.WORKLOAD == "node-density-heavy" ) {
                 parameter_to_pass += "," + NODE_COUNT
             }
-            build job: 'scale-ci/e2e-benchmarking-multibranch-pipeline/write-scale-ci-results', parameters: [string(name: 'BUILD_NUMBER', value: BUILD_NUMBER),text(name: "ENV_VARS", value: ENV_VARS),string(name: 'CI_JOB_ID', value: BUILD_ID), string(name: 'CI_JOB_URL', value: BUILD_URL), string(name: 'JENKINS_AGENT_LABEL', value: JENKINS_AGENT_LABEL), string(name: "CI_STATUS", value: "${status}"), string(name: "JOB", value: WORKLOAD), string(name: "JOB_PARAMETERS", value: "${parameter_to_pass}" ), text(name: "JOB_OUTPUT", value: "${output}")]
+
+            build job: 'scale-ci/e2e-benchmarking-multibranch-pipeline/write-scale-ci-results', parameters: [
+            string(name: 'BUILD_NUMBER', value: BUILD_NUMBER),text(name: "ENV_VARS", value: ENV_VARS),
+            string(name: 'CI_JOB_ID', value: BUILD_ID), string(name: 'CI_JOB_URL', value: BUILD_URL),
+            string(name: 'JENKINS_AGENT_LABEL', value: JENKINS_AGENT_LABEL), string(name: "CI_STATUS", value: "${status}"),
+            string(name: "JOB", value: WORKLOAD), string(name: "JOB_PARAMETERS", value: "${parameter_to_pass}" ),
+            text(name: "JOB_OUTPUT", value: "${output}")], propagate: false
```

Just a formatting suggestion that might help readability, but otherwise LGTM
```suggestion
            build job: 'scale-ci/e2e-benchmarking-multibranch-pipeline/write-scale-ci-results', parameters: [
                string(name: 'BUILD_NUMBER', value: BUILD_NUMBER),
                text(name: "ENV_VARS", value: ENV_VARS),
                string(name: 'CI_JOB_ID', value: BUILD_ID),
                string(name: 'CI_JOB_URL', value: BUILD_URL),
                string(name: 'JENKINS_AGENT_LABEL', value: JENKINS_AGENT_LABEL),
                string(name: "CI_STATUS", value: "${status}"),
                string(name: "JOB", value: WORKLOAD),
                string(name: "JOB_PARAMETERS", value: "${parameter_to_pass}" ),
                text(name: "JOB_OUTPUT", value: "${output}")
            ],
            propagate: false
```